packages/lib-classifier/src/plugins/drawingTools/experimental/models/tools/TranscriptionLineTool/TranscriptionLineTool.js

```diff
@@ -0,0 +1,21 @@
+import { types } from 'mobx-state-tree'
+import { Tool } from '@plugins/drawingTools/models/tools'
+import { TranscriptionLine } from '../../marks'
+
+const TranscriptionLineTool = types.model('TranscriptionLine', {
+  marks: types.map(TranscriptionLine),
+  type: types.literal('polygon')
```

Comment:
I think we need to distinguish this from polygon, even with the similarities. `transcriptionLine`?

Reply:
Sounds good to me. This is only here because my test workflow uses `polygon` in the project builder.

Reply:
```suggestion
  type: types.literal('transcriptionLine')
```